PyTorch/vLLM_Tutorials/FP8_Quantization_using_INC/FP8_Quantization_using_INC.ipynb

```diff
@@ -0,0 +1,516 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set the required environment variables before starting the server. Most importantly, point the `QUANT_CONFIG` variable to the full path quantization config .json as seen above. In this example, it is assumed to be `/root/vllm-hpu-extension/calibration/g2/meta-llama-3.1-70b-instruct/maxabs_quant_g2.json` for a Gaudi 2 machine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%env QUANT_CONFIG=/root/vllm-hpu-extension/calibration/g2/meta-llama-3.1-70b-instruct/maxabs_quant_g2.json\n",
+    "\n",
+    "%env PT_HPU_ENABLE_LAZY_COLLECTIVES=true\n",
+    "%env PT_HPUGRAPH_DISABLE_TENSOR_CACHE=1\n",
+    "%env VLLM_ENGINE_ITERATION_TIMEOUT_S=3600\n",
+    "%env VLLM_PROMPT_BS_BUCKET_MAX=16\n",
+    "%env VLLM_DECODE_BS_BUCKET_MAX=128\n",
+    "%env VLLM_DECODE_BLOCK_BUCKET_MIN=2048\n",
+    "%env VLLM_DECODE_BLOCK_BUCKET_MAX=4096\n",
+    "%env VLLM_PROMPT_SEQ_BUCKET_MAX=2048\n",
+    "%env VLLM_PROMPT_SEQ_BUCKET_MIN=2048\n",
+    "%env VLLM_SKIP_WARMUP=\"true\"\n",
+    "!cd /root/vllm-fork"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use this special start command to launch the vllm server. Note the use of `--quantization inc` and `--kv-cache-dtype fp8_inc` parameters that enable the FP8 quantization using INC and above `QUANT_CONFIG`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "command=f\"python -m vllm.entrypoints.openai.api_server \\\n",
+    "    --port 8080 \\\n",
+    "    --model {CHECKPOINT_PATH} \\\n",
+    "    --tensor-parallel-size 8 \\\n",
+    "    --max-num-seqs 128 \\\n",
+    "    --disable-log-requests \\\n",
+    "    --dtype bfloat16 \\\n",
+    "    --block-size 128 \\\n",
+    "    --gpu-memory-util 0.9 \\\n",
+    "    --num-lookahead-slots 1 \\\n",
+    "    --use-v2-block-manager \\\n",
+    "    --max-num-batched-tokens 32768 \\\n",
+    "    --max-model-len 4096 \\\n",
+    "    --quantization inc \\\n",
+    "    --kv-cache-dtype fp8_inc \\\n",
+    "    --weights-load-device cpu \\\n",
+    "    2>&1 | tee server_70b_TP8_fp8.log\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Launch vLLM server instance:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "process = subprocess.Popen(command, shell=True, start_new_session=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)\n",
+    "print(\"Process ID for server:\", process.pid)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The server process would have launched in the background. Note the process ID for reference. The output logs of the server can can be seen in real-time in the ```server_70b_TP8_fp8.log``` file."
```

Comment:
You may want to specify the actual path that this log file is in. I got: /root/Gaudi-tutorials/PyTorch/vLLM_Tutorials/FP8_Quantization_using_INC/server_70b_TP8_fp8.log

Reply:
Re-running the test, I found we were in the correct directory. Modification is not mandatory.